frontend/src/__mocks__/mockHardwareProfile.ts

```diff
@@ -77,5 +89,6 @@ export const mockHardwareProfile = ({
     tolerations,
     nodeSelectors,
     description,
+    warning,
```

Comment:
Does the CRD have a warning field? If so, this feels too closer to controller land and not really what a spec should be managing 🤔